Cargo.toml

```diff
@@ -2,6 +2,7 @@
 resolver = "2"
 members = [
     "docstrings",
+    "examples/build-all-examples",
```

Comment:
Hmmm. Should this not be in the workspace?

Reply:
This should be fine - the `build-all-examples` "script" itself has no direct crate dependencies so shouldn't cause any conflicts. Actually *running* the script *does* expect there to be an instance of `forc` available in `PATH`, but this doesn't affect the workspace.

The idea behind `build-all-examples` is something like this: After all the existing CI passes, we `cargo install --path ./forc` so that its available via `PATH`. Then we `cargo run --bin build-all-examples` which will use the installed `forc` instance to build each example individually.

Reply:
Ah I see, and `cargo run --bin build-all-examples` requires `build-all-examples` to be built, which is done automagically if it's in the workspace, right?

Reply:
Yeah exactly.

Looks like it's working! We can see the "Build all examples" CI step fail [here](https://github.com/FuelLabs/sway/runs/5142599565?check_suite_focus=true#step:13:101), where `forc build` fails to compile the `wallet_smart_contract` example as `storage` isn't yet implemented. Hopefully this helps to keep the book examples up to date.

Next step is to work out how to run `forc test` for each example too. This is a bit trickier as, unlike `forc build`, `forc test` invokes `cargo test` under the hood and in this case it's happening *inside* the sway workspace, so we need to find a way to get this cargo invocation to "ignore" the sway workspace. Will have a look at this tomorrow!